node/src/cli.rs

```diff
@@ -57,6 +57,14 @@ pub enum Subcommand {
 	/// The custom benchmark subcommmand benchmarking runtime pallets.
 	#[structopt(name = "benchmark", about = "Benchmark runtime pallets.")]
 	Benchmark(frame_benchmarking_cli::BenchmarkCmd),
+
+	/// Try some command against runtime state.
+	#[cfg(feature = "try-runtime")]
+	TryRuntime(try_runtime_cli::TryRuntimeCmd),
+
+	/// Try some command against runtime state. Note: `try-runtime` feature must be enabled.
+	#[cfg(not(feature = "try-runtime"))]
+	TryRuntime,
```

Comment:
Why we need this one? Seems not that necessary

Reply:
> Why we need this one? Seems not that necessary

I copied this from substrate codes..
I think you will get the warning message in `command.rs` if you try to run `try-runtime` while not compiling it with this feature.